test/options/lines-between-rulesets-css/test.js

```diff
@@ -0,0 +1,24 @@
+/* global describe, it */
+'use strict';
+
+describe('options/lines-between-rulesets:css', function() {
```

Comment:
@ravasthi Hi. To be honest I took up this PR so that we could have the fix as soon as possible. Now two months have passed. If you'd like to continue it is ok, personally I've switched to another tool to format the CSS in our projects.